JustSaying/Fluent/ServiceProviderResolver.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using JustSaying.Messaging.MessageHandling;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace JustSaying.Fluent
+{
+    /// <summary>
+    /// A class that implements <see cref="IServiceResolver"/> and <see cref="IHandlerResolver"/>
+    /// for <see cref="IServiceProvider"/>. This class cannot be inherited.
+    /// </summary>
```

Comment:
Are these autogenerated out of interest? There's nothing in this doc comment that isn't in the type definition

Reply:
No, I wrote it, but for public shipping libraries I’m a believer in adding _something_.

Reply:
As-per #414. Otherwise it’ll just generate compiler warnings.